docs/getting-started/concepts.md

```diff
@@ -3,19 +3,33 @@ template: main.html
 ---
 
 # Iter8
-Iter8 is a metrics-driven release optimizer built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well, are secure, and maximize business value.
+Iter8 is the Kubernetes testing platform built for DevSecOps, MLOps, SRE and data science teams. Iter8 makes it easy to ensure that new versions of apps and ML models perform well and maximize business value.
 
-## Iter8 experiment
+## Iter8 Experiment
 <p align='center'>
   <img alt-text="load-test-http" src="../../images/iter8-intro-dark.png" width="70%" />
 </p>
 
-Iter8 experiments make it simple to collect performance, risk, and business metrics for apps and ML models, assess, compare and validate one or more app/ML model versions, promote the winning version, and maximize business value in each release.
+Iter8 experiments makes it simple to collect performance and business metrics for apps and ML models, assess and compare one or more app/ML model versions, validate [service-level objectives (SLOs)](#service-level-objectives), promote the winning version, and maximize business value during each release.
 
-## Experiment chart
-Iter8 experiment charts are [Helm charts](https://helm.sh/docs/topics/charts/) under the covers, and enable simple, declarative, and reusable metrics-driven experiments. Iter8 combines experiment charts with user supplied values to generate runnable `experiment.yaml` files or Kubernetes experiment manifests. The former is used for running experiments in the local environment, while the latter is used for running experiments inside Kubernetes.
+Specifying an Iter8 experiment involves specifying the list of [tasks](#tasks) executed during the experiment, their [parameters](../user-guide/topics/parameters.md), and the experiment [runner](#loops-and-runner).
+
+### Tasks
+An experiment is an ordered collection of tasks that are executed sequentially. Iter8 provides pre-defined tasks for various functions such as generating load and collecting Iter8's built-in metrics for HTTP and gRPC services, collecting custom metrics for one or more versions of an app from databases, assessing [SLOs](#service-level-objectives), and checking if the application is ready.
```

Comment:
```suggestion
An experiment is an ordered collection of tasks that are executed sequentially. Iter8 provides pre-defined tasks for various functions such as generating load and collecting Iter8's built-in metrics for HTTP and gRPC services, collecting custom metrics from databases, assessing [SLOs](#service-level-objectives), and checking if the application is ready.
```

This is a pretty long sentence so I was wondering if we could shorten it a bit.

Reply:
Fixed.